mne/label.py

```diff
@@ -1973,22 +1992,21 @@ def read_labels_from_annot(subject, parc='aparc', hemi='both',
     if regexp is not None:
         # allow for convenient substring match
         r_ = (re.compile('.*%s.*' % regexp if regexp.replace('_', '').isalnum()
-              else regexp))
+                         else regexp))
 
     # now we are ready to create the labels
     n_read = 0
     labels = list()
     for fname, hemi in zip(annot_fname, hemis):
         # read annotation
         annot, ctab, label_names = _read_annot(fname)
-        label_rgbas = ctab[:, :4]
+        label_rgbas = ctab[:, :4] / 255.
         label_ids = ctab[:, -1]
 
         # load the vertex positions from surface
-        fname_surf = op.join(subjects_dir, subject, 'surf',
-                             '%s.%s' % (hemi, surf_name))
-        vert_pos, _ = read_surface(fname_surf)
-        vert_pos /= 1e3  # the positions in labels are in meters
+        vert_pos = _load_vert_pos(
+            subject, subjects_dir, surf_name, hemi, len(annot),
+            extra='for annotation file %s' % fname)
```

Comment:
`extra='for annotation file ' + fname)`?

Reply:
Don't think it matters. I usually do `%s` and `%` anyway because sometimes I'll end up adding other things, and it's already in the format to do so